docker-compose.yml

```diff
@@ -1,16 +1,5 @@
 version: '2'
 services:
-  kong-frontend:
```

Comment:
@koudaiii Fixed this and I've confirmed in my local env.

![1 docker-compose up docker-compose 2016-09-21 21-36-01](https://cloud.githubusercontent.com/assets/7839872/18711188/b1d90834-8043-11e6-835b-2cb480755cf1.png)